new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioTwoDomainsInImage.java

```diff
@@ -0,0 +1,367 @@
+        replicaCount);
+
+    logger.info("Check for domain custom resource in namespace {0}", domainNamespace1);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                domainUid1,
+                domainNamespace1,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(domainExists(domainUid1, DOMAIN_VERSION, domainNamespace1));
+
+    logger.info("Check for domain custom resource in namespace {0}", domainNamespace2);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                domainUid2,
+                domainNamespace2,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(domainExists(domainUid2, DOMAIN_VERSION, domainNamespace2));
+
+    // check admin server pod is ready
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName1, domainNamespace1);
+    checkPodReady(adminServerPodName1, domainUid1, domainNamespace1);
+
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName2, domainNamespace2);
+    checkPodReady(adminServerPodName2, domainUid2, domainNamespace2);
+
+    // check managed server pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed pod {0} to be ready in namespace {1}",
+          managedServerPrefix1 + i, domainNamespace1);
+      checkPodReady(managedServerPrefix1 + i, domainUid1, domainNamespace1);
+      logger.info("Wait for managed pod {0} to be ready in namespace {1}",
+          managedServerPrefix2 + i, domainNamespace2);
+      checkPodReady(managedServerPrefix2 + i, domainUid2, domainNamespace2);
+    }
+
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName1, domainNamespace1);
+    checkServiceExists(adminServerPodName1, domainNamespace1);
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName2, domainNamespace2);
+    checkServiceExists(adminServerPodName2, domainNamespace2);
+
+    // check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managedserver service {0} is created in namespace {1}",
+          managedServerPrefix1 + i, domainNamespace1);
+      checkServiceExists(managedServerPrefix1 + i, domainNamespace1);
+      logger.info("Check managedserver service {0} is created in namespace {1}",
+          managedServerPrefix2 + i, domainNamespace2);
+      checkServiceExists(managedServerPrefix2 + i, domainNamespace2);
```

Comment:
The podReady and serviceExists should be done in this order.
1. check admin service exists
2. check admin pod ready
3. check ms service exists
4. check ms pod ready

Reply:
done